azure-pipelines.yml

```diff
@@ -147,6 +147,10 @@ jobs:
       name: NetCoreInternal-Int-Pool
```

Comment:
Small Nit: This job no longer builds the packages, should we change the display name to Sign Artifacts?

Reply:
Sorry - I merged before I saw this. I will do this next time I'm in here.

Reply:
No problem. I approved before realizing the name was off 😄